### PR TITLE
2024 04 03 TAC

### PR DESCRIPTION
**Pull request by ryjones**

### CODE REVIEW

**Review by brian-jarvis-aws — APPROVED** [review]

File: meeting-minutes/2024/2024-04-03-TAC.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: 2024-04-03 TAC Meeting Record
```
Comment (ghost) [inline meeting-minutes/2024/2024-04-03-TAC.md:3]: Should we use a non unambiguous date format here? We could use the `date` format as defined in [RFC 2822 § 3.3]:

```
date            =       day month year
day             =       ([FWS] 1*2DIGIT) / obs-day
month           =  "Jan"  /  "Feb" /  "Mar"  /  "Apr"
                /  "May"  /  "Jun" /  "Jul"  /  "Aug"
                /  "Sep"  /  "Oct" /  "Nov"  /  "Dec"
year            =       4*DIGIT / obs-year
```

So for instance here: `3 Apr 2024`

[RFC 2822 § 3.3]: https://www.rfc-editor.org/rfc/rfc2822.txt

Comment (brian-jarvis-aws) [reply meeting-minutes/2024/2024-04-03-TAC.md:3]: The existing format is following ISO 8601, at least the YYYY-MM-DD portion of it, which I prefer. It better lends itself to chronological sorting.

Comment (brian-jarvis-aws) [reply meeting-minutes/2024/2024-04-03-TAC.md:3]: If you're talking about just the way it is displayed, I have no strong preference. But as written it is consistent with the date formatting in the filename.

Comment (ghost) [reply meeting-minutes/2024/2024-04-03-TAC.md:3]: Okay, sgtm then. +1 for the chronological sorting

**Review by ghost — APPROVED** [review]

File: meeting-minutes/2024/2024-04-03-TAC.md (hunk 1 above)
Comment (ryjones) [reply meeting-minutes/2024/2024-04-03-TAC.md:3]: Personally, I would prefer 03 APR 2024, but 2024-04-03 does make sorting much easier 😀

**Comment by ryjones** [comment]
@thb-sb @brian-jarvis-aws please merge when ready

**Comment by brian-jarvis-aws** [comment]
Write access needed to be able to merge

**Comment by ryjones** [comment]
@brian-jarvis-aws oversight on my part. [should work now.](https://github.com/PQCA/governance/pull/9/files)

**Pull request merged**